iceoryx_hoofs/include/iceoryx_hoofs/log/logging.hpp

```diff
@@ -22,14 +22,24 @@ namespace iox
 {
 namespace log
 {
+namespace internal
+{
+/// @brief Convenience function for the IOX_LOG_INTERNAL macro
+inline bool isLogLevelActive(LogLevel logLevel)
```

Comment:
hahaha, you did spot it. I did not want to create a new file only for this single return statement. Additionally, it was nice to have it just above the macro to immediately tell when the log message is active. Since you now know my intention, would you permit an exception?

I won't argue if you still like this to be moved to an `inl` file.

Reply:
Lets see if there are some static code analyzers complaining and then you have to do it either way or write a suppression. But do as you please.